### PR TITLE
Std.is to Std.isOfType in Manager.hx and getter/setter macro path in RecordMacros.hx

### PR DESCRIPTION
**Pull request by filt3rek**
Hej @jonasmalacofilho ,

After 1 year I finally made a PR as you told me. These are minor fixes, if you could take a look when you have time and tell me if it's ok for you ?
Thanks for your patience, sorry for this delay.

Regards,

### CODE REVIEW

**Comment by jonasmalacofilho** [comment]
Hi @filt3rek!

I'm sorry, but it's been quite a while since I worked with Haxe: while the changes here seem reasonable to me, I can't immediately remember _why_ they are necessary.

(As a general suggestion, try to split unrelated changes into different PRs, or at least different commits; and try to explain why each of them is necessary, what's the reasoning for that specific fix, if there are any backward compatibility concerns, etc).

I also no longer have a local Haxe+targets environment set up to test this, and it seems that the repository is still only set up to use Travis CI, which doesn't run anymore.

TL;DR: I'm not the right person to review/merge this anymore, sorry!

**Comment by filt3rek** [comment]
Thx for your reply !

Ok, I'll try to add more informations.

And if someone will take this PR in the future it's ok (for my needs I have all what I want :stuck_out_tongue_closed_eyes: 

BTW I don't know why you don't work with Haxe anymore, I hope everything is going fine for you.

Good luck !

**Comment by jonasmalacofilho** [comment]
> BTW I don't know why you don't work with Haxe anymore, I hope everything is going fine for you.
> Good luck !

Yes, everything is fine, thanks!  Just working on slightly different things and/or with different tools.

**Review by filt3rek — COMMENTED** [review]
Sorry I've not added enough infos first time, so I put it here, I don't know if it is the right place but I try.

In Manager.hx, the only changes are with `Std.is` to `Std.isOfType` just to avoid compiler warnings

Then in RecordMacros.hx the changes concerns the getter and setter generations and all is well explained (I hope) here : https://github.com/HaxeFoundation/record-macros/issues/57#issuecomment-751342817

Regards,

EDIT : arf, I think it's not the best place to add the infos, I don't know where add them so, sorry.

**Comment by logo4poop** [comment]
@ncannasse I know this isn't your project, but I assume you have merge permissions here. Not a big change or anything, but is nice

**Comment by mockey** [comment]
Hey @filt3rek, are you still working with record-macros? You seem to be the only one that made some PRs here recently. I have an old project with record-macros, that does not compile with latest haxe. I applied your patches, now I get the error:
`Class<data.Version> has no field manager`
where data.Version extends sys.db.Object. Any idea?
I saw that you commented out the @:autoBuild in sys.db.Object. Does macroBuild have to run in another place then maybe?

**Comment by filt3rek** [comment]
Hej @mockey !

Yes I still use record-macros, I just don't know why the commit "39eeb9f" is here. I use it personnally because of a conflict with `haxe.macro.Compiler.setFieldType` but you don't need that.
So to be clear, you can apply all my commits but not the last one (which removes `@:autoBuild(sys.db.RecordMacros.macroBuild())`) and all should work fine for you.

**Comment by mockey** [comment]
Hey @filt3rek.
So I should go without the whole commit or just the @:autoBuild removal? You made some changes in RecordMacros there as well.
Thanks for the other PRs BTW. I'm glad that at least one person is still caring about this.
On a sidenote: All my compilation errors where coming from the vshaxe compilation server. Adding `"haxe.enableCompilationServer": false` to settings.json fixes this. Now it compiles without a problem. I remember I already had this some time ago with another project. vshaxe seems to have problems with heavy macro code.

**Comment by filt3rek** [comment]
You should just remove the last commit (where I commented `@:autoBuild`)
